{"name": "test_signin_api_using_invalid_specialCharacterValue_phone", "status": "failed", "statusDetails": {"message": "AssertionError: assert 200 == 400\n +  where 200 = <Response [200]>.status_code", "trace": "self = <Server.API.Test.test_signup_trado_commerce_api.TestLogin_API object at 0x00000266A9055610>\n\n    @allure.description('test_signin_api_using_invalid_specialCharacterValue_phone')\n    def test_signin_api_using_invalid_specialCharacterValue_phone(self):\n        url = SignupConstants.url\n        data = SignupConstants.invalid_specialCharacterValue_phone\n        res = requests.post(url, data)\n        res_data = res.json()\n>       assert res.status_code == 400\nE       assert 200 == 400\nE        +  where 200 = <Response [200]>.status_code\n\ntest_signup_trado_commerce_api.py:61: AssertionError"}, "description": "test_signin_api_using_invalid_specialCharacterValue_phone", "start": 1677093684543, "stop": 1677093685039, "uuid": "d75f9926-6345-4f8c-81c4-55dbed558e15", "historyId": "5136e075fc8e86b6a209a1bc1dd6b9b8", "testCaseId": "2402cb55a023ab2234b3a05c930e6276", "fullName": "test_signup_trado_commerce_api.TestLogin_API#test_signin_api_using_invalid_specialCharacterValue_phone", "labels": [{"name": "suite", "value": "test_signup_trado_commerce_api"}, {"name": "subSuite", "value": "TestLogin_API"}, {"name": "host", "value": "LAPTOP-CFNBSI63"}, {"name": "thread", "value": "12920-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_signup_trado_commerce_api"}]}